core/src/impl/KokkosExp_IterateTileGPU.hpp

```diff
@@ -66,97 +66,42 @@ struct EmulateCUDADim3 {
 };
 #endif
 
+template <class Tag, class Functor, class... Args>
+KOKKOS_IMPL_DEVICE_FUNCTION
+    typename std::enable_if<std::is_void<Tag>::value>::type
+    _tag_invoke(Functor const& f, Args&&... args) {
+  f((Args &&) args...);
+}
+
+template <class Tag, class Functor, class... Args>
+KOKKOS_IMPL_DEVICE_FUNCTION
+    typename std::enable_if<!std::is_void<Tag>::value>::type
```

Comment:
```suggestion
    std::enable_if_t<!std::is_void<Tag>::value>
```

Reply:
Done in [`5e59404` (#3624)](https://github.com/kokkos/kokkos/pull/3624/commits/5e5940405575404039658fa8a5c55724d7a2dccd).